ERCS/erc20-holder-extension-for-nfts.md

```diff
@@ -0,0 +1,179 @@
+---
```

Comment:
```suggestion
---
eip: 7590
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Perfect, thanks!